tests/parser/functions/test_rlp_list.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+from tests.setup_transaction_tests import chain as s, tester as t, ethereum_utils as u, check_gas, \
+    get_contract_with_gas_estimation
+
+def test_rlp_decoder_code():
+    import rlp
```

Comment:
why the inline import? Isn't rlp a package ?

Reply:
You're right, I think there's no benefit to having it inline.